Include/py_curses.h

```diff
@@ -64,6 +62,13 @@ typedef struct {
     char *encoding;
 } PyCursesWindowObject;
 
+typedef struct {
+    PyTypeObject *window_type;
+    int (*setup_term_called)(void);
+    int (*initialized)(void);
+    int (*initialized_color)(void);
+} PyCurses_CAPI;
```

Comment:
Please make it private:

```suggestion
} _PyCurses_CAPI;
```